src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends AbstractFigure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.